### DISCUSSION TITLE
[SOLVED] Just installed on Android 14 and got Japanese UI, help!

### DISCUSSION THREAD

**Discussion by janaxhell** [bitfireAT/icsx5#242] [Q&A] [answered]
I've just bought ICSx5 from Google Playstore and as soon as it installed the UI came up in Japanese. I have my phone set to Italian and I cannot find any setting in the app to change it so. Please help, I have no idea what the UI is telling me.

EDIT Ok, I just forced the language from Apps Settings in Android (which was set to system default, hence Italian) and now I can read it. But why did it set itself to Japanese?

EDIT 2 More weird: Simple Calendar shows the Webcal slot from ICSx5 with a Japanese title.

**Comment by ArnyminerZ** [comment, marked as answer]
Looks like a misconfiguration in your device. Can you make sure Japanese is not added as a secondary language in your system's configuration? ICSx5 is currently not available in Italian, so if you have it set as a secondary language, it might fall back to it.

By the way, if you want to contribute with Italian translations, we have [a page on Transifex](https://explore.transifex.com/bitfireAT/icsx5/) for that 😉

**Comment by janaxhell** [comment]
Yes, correct, I've bought my phone used and I see now it has Italian, English and Japanese set as languages. For some reason your app prefers to fallback to Japanese.

**Reply by ArnyminerZ** [reply]
Language fallbacks in Android are really strange, they work kind of when they want, so no surprise it's selecting a random one.
